server/service/IndividualServicesService.js

```diff
@@ -63,69 +63,68 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
-         }
-         if (update.isReleaseUpdated || update.isApplicationNameUpdated) {
-           let configurationStatus = new ConfigurationStatus(
-             newReleaseHttpClientLtpUuid,
-             undefined,
-             true);
-          
-             logicalTerminationPointConfigurationStatus.httpClientConfigurationStatus = configurationStatus;
-             
-           }
-         
-           let newReleaseTcpClientUuidList = await logicalTerminationPoint.getServerLtpListAsync(newReleaseHttpClientLtpUuid);
-           let newReleaseTcpClientUuid = newReleaseTcpClientUuidList[0];
-           if(protocol != currentNewReleaseRemoteProtocol){
-             update.isProtocolUpdated = await tcpClientInterface.setRemoteProtocolAsync(newReleaseTcpClientUuid, applicationProtocol);
-           }
-           if(address != currentNewReleaseRemoteAddress) {
-             update.isAddressUpdated = await tcpClientInterface.setRemoteAddressAsync(newReleaseTcpClientUuid, applicationAddress);
-           }
-           if(port != currentNewReleaseRemotePort) {
-             update.isPortUpdated = await tcpClientInterface.setRemotePortAsync(newReleaseTcpClientUuid, applicationPort);
-           }
-   
-           let isdataTransferRequired = true;
-           let serverAddress = await tcpServerInterface.getLocalAddressOfTheProtocol(applicationProtocol);
-            let serverPort = await tcpServerInterface.getLocalPortOfTheProtocol(applicationProtocol);
-             if ( address === serverAddress && port === serverPort) {
-               isdataTransferRequired = false;
-                    }
-   
-           if (update.isProtocolUpdated || update.isAddressUpdated || update.isPortUpdated) {
-             let configurationStatus = new ConfigurationStatus(
-               newReleaseTcpClientUuid,
-               undefined,
-               true);
-             logicalTerminationPointConfigurationStatus.tcpClientConfigurationStatusList = [configurationStatus];
-             
-           }
-           if (logicalTerminationPointConfigurationStatus != undefined) {
-   
+        }
+        if (applicationName != currentNewReleaseApplicationName) {
+          update.isApplicationNameUpdated = await httpClientInterface.setApplicationNameAsync(newReleaseHttpClientLtpUuid, applicationName);
+        }
+        if (update.isReleaseUpdated || update.isApplicationNameUpdated) {
+          let configurationStatus = new ConfigurationStatus(
+            newReleaseHttpClientLtpUuid,
+            undefined,
+            true);
+
+          logicalTerminationPointConfigurationStatus.httpClientConfigurationStatus = configurationStatus;
+
+        }
+
+
+        if (protocol != currentNewReleaseRemoteProtocol) {
+          update.isProtocolUpdated = await tcpClientInterface.setRemoteProtocolAsync(newReleaseTcpClientUuid, protocol);
+        }
+        if (address["ip-address"]["ipv-4-address"] !== currentNewReleaseRemoteAddress["ip-address"]["ipv-4-address"]) {
```

Comment:
domain-name can be present and has to be checked

Reply:
done